includes/legacy/class-wc-legacy-cart.php

```diff
@@ -58,9 +58,18 @@ abstract class WC_Legacy_Cart {
 	 * @param mixed  $value Value to set.
 	 */
 	public function __isset( $name ) {
-		if ( array_key_exists( $name, $this->cart_session_data ) || 'fees' === $name ) {
+		$legacy_keys = array_merge(
+			array(
+				'tax_display_cart',
+				'fees',
+			),
```

Comment:
Just curious, why doesn't this array include all the keys that are handled in `__get`?